hooks/persistence-dependencytrack/Chart.yaml

```diff
@@ -0,0 +1,30 @@
+# SPDX-FileCopyrightText: the secureCodeBox authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+apiVersion: v2
+name: persistence-dependencytrack
+description: The dependencytrack persistence provider persists secureCodeBox CycloneDX SBOMs into Dependency-Track.
+
+type: application
+
+# version - gets automatically set to the secureCodeBox release version when the helm charts gets published
+version: "4.0.0"
+
+# TODO does it make sense to set this? DefectDojo's seems quite outdated, docs say set this and version to latest
+appVersion: "4.8.2"
```

Comment:
The docs said to set this value, but I'm not sure how much sense it makes, I compared it to the DefectDojo hook and the value there is quite outdated. Probably it just needs to be here? This is the current Dependency-Track version that I tested against, but not the version of the hook.

Reply:
I guess for keeping it similar to the other hooks I'll leave it in here for now, but I opened #1959 to keep track of this.